agent/e2e/agent-tui/test-no-registry-agent-tui.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../../../" && pwd )"
+source $SCRIPTDIR/common.sh
+source $SCRIPTDIR/agent/e2e/agent-tui/utils.sh
+
+set +x
+shopt -s nocasematch
+
+node_name=$1
+
+# The following commands sends key presses through "virsh send-key" to interact
+# with agent-tui
+if [[ "$node_name" == "${CLUSTER_NAME}_master_0" ]]; then
```

Comment:
This works, but I hoped for a more robust way to identify the rendezvous node (given that is an arbitrary decision, it could be stored in a RENDEZVOUS_NODE_HOSTNAME for example)